Mathlib/Combinatorics/SimpleGraph/Triangle/Basic.lean

```diff
@@ -38,50 +36,49 @@ namespace SimpleGraph
 variable {α 𝕜 : Type*} [Fintype α] [LinearOrderedField 𝕜] {G H : SimpleGraph α} {ε δ : 𝕜} {n : ℕ}
   {s : Finset α}
 
-/-- A simple graph is *`ε`-triangle-free far* if one must remove at least `ε * (card α)^2` edges to
```

Comment:
This should probably say `ε`-far from triangle-free

Reply:
All done.